chart/values.yaml

```diff
@@ -39,11 +39,30 @@ gitvote:
   addr: 0.0.0.0:9000
 
   # GitHub configuration
+  #
+  # The app require the following permissions:
+  # - Repository permissions:
+  #   - Checks: read/write
+  #   - Contents: read
+  #   - Issues: read/write
+  #   - Metadata: read
+  #   - Pull requests: read/write
+  # - Organization
+  #   - Members: read
+  #
+  # Subscribe to events:
+  # - Issue Comment
```

Comment:
There are a couple of events to subscribe missing, could you please add them?

- Issues
- Pull request